eqc/pbkdf2_eqc.erl

```diff
@@ -0,0 +1,82 @@
+-module(pbkdf2_eqc).
```

Comment:
I'm assuming this file is left untouched except for the occasional static analysis -related fix. Is my assumption correct?

Reply:
In the sense that we won't run the tests contained within it? Sure.